crates/bitwarden/src/auth/renew.rs

```diff
@@ -63,8 +63,10 @@ pub(crate) async fn renew_token(client: &mut Client) -> Result<()> {
                     ) = (&result, state_file, client.get_encryption_settings())
                     {
                         if let Some(enc_key) = enc_settings.get_key(&None) {
-                            let state =
-                                ClientState::new(r.access_token.clone(), enc_key.to_base64());
+                            let state = ClientState::new(
```

Comment:
Should state expect sensitive values directly?

Reply:
Just checked and it's not too big a diff, changed it